rhel/internal/common/updater.go

```diff
@@ -38,6 +39,7 @@ func NewUpdater(url string, init interface{}) *Updater {
 		reqRate: rate.NewLimiter(interval, 1),
 	}
 	u.value.Store(init)
+	u.firstUse.Store(true)
 	// If we were provided an initial value, pull the first token.
 	if !reflect.ValueOf(init).IsNil() {
 		u.reqRate.Allow()
```

Comment:
Couldn't this same functionality be achieved by not pulling a token here? Am I missing something obvious?